src/modules/core/components/Popover/Popover.tsx

```diff
@@ -116,31 +120,42 @@ const Popover = ({
 
   const lastIsOpenProp = usePrevious(isOpenProp);
 
+  const close = useCallback(
+    (data?: any, modifiers?: { cancelled: boolean }) => {
+      if (openTimeoutRef.current) {
+        clearTimeout(openTimeoutRef.current);
+      }
+      if (closeTimeoutRef.current) {
+        clearTimeout(closeTimeoutRef.current);
+      }
+      _setIsOpen(false);
+      if (typeof onClose == 'function') onClose(data, modifiers);
+    },
+    [onClose],
+  );
+
+  useEffect(() => {
+    if (closeAfterDelay) {
+      closeTimeoutRef.current = setTimeout(() => {
+        close();
+      }, 3000 + (openDelay || 0));
```

Comment:
Why is this value hardcoded ? This should be a prop!

Reply:
oops, I added a prop for this now.